README.md

```diff
@@ -360,3 +360,16 @@ $ out/clang/Debug/fuzz/wasm2wat_fuzz ...
 
 See the [libFuzzer documentation](https://llvm.org/docs/LibFuzzer.html) for
 more information about how to use this tool.
+
+## Installing prebuilt binaries
+
+Wabt is available on many platforms as prepackaged binaries. For example if you use Homebrew you can use:
```

Comment:
Can you add a comma after "For example" , and wrap this line at 80 columns (same with the very last line)